ci-operator/step-registry/cucushift/installer/rehearse/gcp/upi/ovn/sdn2ovn/provision/cucushift-installer-rehearse-gcp-upi-ovn-sdn2ovn-provision-chain.yaml

```diff
@@ -0,0 +1,11 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-upi-ovn-sdn2ovn-provision
+  steps:
+  - ref: ipi-install-rbac
+  - chain: upi-conf-gcp
+  - ref: upi-install-gcp
+  - ref: sdn-conf
+  - ref: ipi-install-times-collection
+  - ref: enable-qe-catalogsource
```

Comment:
Here no migration step?

Reply:
> Here no migration step?

@zhaozhanqi Good catch! Just add it. Thanks!